### ISSUE TITLE
Ctrl-click on item to go to its RealmEye page

### ISSUE THREAD

**Issue by AdityaGupta1** [atomizer/muledump#100]
I think it would be cool if ctrl-clicking on an item takes you to its RealmEye page.

The link to the item's page is usually `http://www.realmeye.com/wiki/<item-name>`, where `<item-name>` is the item's name in all lowercase and with dashes instead of spaces.

For example, the Hallucination Prism (T3) is `http://www.realmeye.com/wiki/hallucination-prism`.

**Comment by UnicoRush** [comment]
I like the idea. Although I think that we could additionaly add links to trade page at RealmEye. Like shift+click and then choose either to buy or sell. 

Links are similar to each other. For selling there is: http://www.realmeye.com/offers-to/sell/<item-id> . For buying http://www.realmeye.com/offers-to/buy/<item-id> . 

For example Prism of Figments (T4) has links like http://www.realmeye.com/offers-to/sell/2847 , http://www.realmeye.com/offers-to/buy/2847 . 

Realmeye does not contain pages for early tiers. Only T0 and then T4+ for abilities. T7+ for armors. T6+ for weapons. T3+ for rings.

xx,
Unicorush

**Comment by AdityaGupta1** [comment]
I like the idea of being able to go to the RealmEye trading page.

Also, there are pages for early tier items on RealmEye - for example, the page for the T1 Robe at [http://www.realmeye.com/wiki/robe-of-the-neophyte](http://www.realmeye.com/wiki/robe-of-the-neophyte).

**Comment by Nightfirecat** [comment]
Should <kbd>Ctrl</kbd> / <kbd>Shift</kbd> + click also highlight the item in question when opening a new tab?

@UnicoRush What exactly did you have in mind for the interface for choosing buy/sell? Like a drop-down menu, or a different shortcut for each? (for example, Shift=buy, Alt=sell)

**Comment by AdityaGupta1** [comment]
I don't think it should be too complicated; if someone wants to highlight the item, they can do it separately.

**Comment by UnicoRush** [comment]
@Jordan I meant a drop-down menu. Too much shortcuts may be confusing for users.

**Comment by Nightfirecat** [comment]
@UnicoRush And how is this drop-down menu brought up? What items would the menu contain?

**Comment by UnicoRush** [comment]
@Jordan Inside I would attach options like ‚Buy this item’ and ‚Sell this item’, which would take user to links like:
 For selling  http://www.realmeye.com/offers-to/sell/<item-id> <http://www.realmeye.com/offers-to/sell/%3Citem-id%3E> . For buying http://www.realmeye.com/offers-to/buy/<item-id> <http://www.realmeye.com/offers-to/buy/%3Citem-id%3E> .
